### DISCUSSION TITLE
prop backgroundColor para o componente IconButton

### DISCUSSION THREAD

**Discussion by vrlopess** [TiendaNube/nimbus-design-system#42] [Ideas] [answered]
É possível adicionar a propriedade backgroundColor para IconButton?

https://tiendanube.github.io/nimbus-design-system/?path=/docs/atomic-icon-button--base

<img width="343" alt="image" src="https://user-images.githubusercontent.com/87648920/196729304-2a9f4734-3a0f-454a-bca8-57190c26074e.png">

<img width="147" alt="image" src="https://user-images.githubusercontent.com/87648920/196729342-0236bae7-4c12-4640-9069-fe4998955ba8.png">

**Comment by jcampos89** [comment, marked as answer]
Oi @vrlopess, obrigado por compartilhar sua dúvida! 

Recentemente atualizamos os tokens de cor para alinhar o visual com o Nimbus V2, possivelmente o design demonstrado acima está desatualizado. A versão com cor correta é a do screenshot que você enviou abaixo e que está dentro do [storybook do Nimbus V2](https://tiendanube.github.io/nimbus-design-system/?path=/docs/atomic-icon-button--base). 

Você pode nos dar mais contexto de onde estamos usando essa cor antiga?

Falando específicamente dessa necessidade de trocar a cor, não vemos motivos para adicionar uma prop, já que seria o caso de atualizar o componente. Faz sentido pra você?

Se tiver alguma outra dúvida, fico a disposição.

**Reply by vrlopess** [reply]
Perfeito! Faz sentido pra mim sim. Acredito que deve estar desatualizado mesmo, porque estou seguindo um design do partners portal que foi feito faz algum tempo. Muito obrigado @jcampos89
